src/main/java/Lotto/model/LottoBuyResource.java

```diff
@@ -0,0 +1,32 @@
+package Lotto.model;
+
+import static Lotto.LottoConstant.LOTTO_PRICE;
+
+public class LottoBuyResource {
+
+    private static final String PRICE_ERROR_MESSAGE = "로또는 한개당 천원입니다.";
+
+    private int lottoPrice;
```

Comment:
초기화를 통해 이미 값이 결정되고 이 후에 변경이 없다면 (변경을 할 수 없게 해야 한다면) `final`로 가시죠 👍 
```suggestion
    private final int lottoPrice;
```